deployments/helm/icmp-responder/templates/icmp-responder-nse.tpl

```diff
@@ -45,4 +45,4 @@ spec:
               networkservicemesh.io/socket: 1
 metadata:
   name: icmp-responder-nse
-  namespace: default
+  namespace: nsm-system
```

Comment:
I guess for the helm charts we should use parameters instead of hardcoding these. Let's take a note and do this in a follow up PR.